tasks/sep/026-fp-holocene-upgrade-fix/README.md

```diff
@@ -0,0 +1,37 @@
+# Holocene Hardfork Upgrade
+
+Status: READY TO SIGN
+
+## Objective
+
+Upgrades the Fault Proof contracts of **OP Sepolia** for Holocene, fixing an upgrade with wrongly newly deployed `DelayedWETH` from #374.
+
+Governance post of the upgrade can be found at https://gov.optimism.io/t/upgrade-proposal-11-holocene-network-upgrade/9313.
+
+This upgrades the Fault Proof contracts in the
+[op-contracts/v1.8.0-rc.4](https://github.com/ethereum-optimism/optimism/tree/op-contracts/v1.8.0-rc.4) release.
+
+## Pre-deployments
+
+- `MIPS` - `0x69470D6970Cd2A006b84B1d4d70179c892cFCE01`
```

Comment:
In this upgrade, is the MIPS contract is used? 
I can't see any transaction or storage modified with it?